Detectors/ITSMFT/MFT/condition/CMakeLists.txt

```diff
@@ -10,14 +10,17 @@
 
 o2_add_library(MFTCondition
                SOURCES src/MFTDCSProcessor.cxx
+                       src/DCSNameResolver.cxx                       
                PUBLIC_LINK_LIBRARIES O2::CCDB
                O2::DetectorsCalibration
                O2::DetectorsDCS
                ROOT::Minuit
+               ROOT::XMLIO
                Microsoft.GSL::GSL)
 
 o2_target_root_dictionary(MFTCondition
                           HEADERS include/MFTCondition/MFTDCSProcessor.h
+                                  include/MFTCondition/DCSNameResolver.h
```

Comment:
since you require dictionary generation for this class, you should add `ClassDefNV(DCSNameResolver,1)` to it.
Also, it should be added to MFTConditionLinkDef. 
I think this is a reason `readMFTDCSentries.C` test fails.